app/pb_kits/playbook/pb_card/_card.jsx

```diff
@@ -36,13 +79,30 @@ const Card = ({
     [`highlight_${highlight.color}`]: highlight.color,
```

Comment:
Should highlights also take category colors? Could highlights be used as categories in some cases?

Reply:
I think highlight can take category colors. There's one _distant_ use case I can think of - course listing on Learning Dojo (admin side). Even though it uses table row side highlight, I can see it working with card side highlight with category colors.

Reply:
Sounds good.

let's chat at dailies...perhaps there's a better way to define the options since our list is really long